README.md

```diff
@@ -99,12 +104,13 @@ will produce firmware file `build/u2f.bin`.
 
 Supported targets are:
 
-- TOMU
-- MAPLE_MINI (STM32F103 board with push button on PB8)
-- BLUE_PILL (generic STM32F103 board without push button)
-- ST_DONGLE (ST-Link V2 clone, push button pulls down PA5/SWCLK pin)
+- [TOMU] (http://tomu.im/)
```

Comment:
it doesn't work because of the space between ] and (
I has to be
```
[TOMU](http://tomu.im/)
```
instead of
```
[TOMU] (http://tomu.im/)
```

Reply:
okk